roles/edpm_logrotate_crond/tasks/configure.yml

```diff
@@ -22,8 +22,6 @@
         path: "/var/lib/openstack"
         mode: 0755
         state: directory
-        owner: "{{ ansible_user }}"
-        group: "{{ ansible_user }}"
```

Comment:
/var/lib/openstack  is not ment to be owned by root
its meant to be owned by the ansible user so this is a bug.